core/files/css/base.php

```diff
@@ -780,7 +787,9 @@ private function get_style_control_value( array $control, array $values ) {
 			return $this->get_selector_global_value( $control, $values['__globals__'][ $control['name'] ] );
 		}
 
-		$value = $values[ $control['name'] ];
+		$value = isset( $values[ $control['name'] ] )
+			? $values[ $control['name'] ]
+			: $this->preserved_dynamic_style_values[ $control['name'] ];
```

Comment:
@louiswol94 You should check isset on `$this->preserved_dynamic_style_values` too. No?

Reply:
You're right, but still, how you can make sure the value is set there?

Reply:
@KingYes Let me add null as a fallback? :)

Reply:
I think it's okay. Thanks.

Reply:
@KingYes Updated.